model.py

```diff
@@ -1969,7 +1969,7 @@ def type_and_identifier_for_urn(cls, identifier_string):
         return (type, identifier_string)
 
     @classmethod
-    def parse_urns(cls, _db, identifier_strings):
+    def parse_urns(cls, _db, identifier_strings, autocreate=True):
```

Comment:
This keeps the default assumptions of `parse_urns` in line with those of `parse_urn`.